src/rpc/RPC.cpp

```diff
@@ -91,6 +91,37 @@ make_HttpContext(
         clientIp};
 }
 
+constexpr static WarningInfo warningInfos[]{
```

Comment:
Does this need to be constexpr?

Reply:
It doesn't for how it is used now, but the warning info is predetermined and therefore can be constexpr